core/isolate.rs

```diff
@@ -690,6 +695,7 @@ impl Future for Isolate {
       drop(locker);
     }
 
+    self.run_microtasks()?;
```

Comment:
Might need to be inside a locker scope.

Reply:
It shouldn't. Lock is being acquired already within `libdeno::deno_run_microtasks`. We only need it on respond, because we don't create a lock in `libdeno::deno_respond`. It might make sense to remove `LockerScope`, and just add a lock to the async part of `libdeno::deno_respond`(We shouldn't need it for sync responses).

Reply:
It shouldn't. Lock is being acquired already within libdeno::deno_run_microtasks. We only need it on respond, because we don't create a lock in libdeno::deno_respond. It might make sense to remove LockerScope, and just add a lock to the async part of libdeno::deno_respond(We shouldn't need it for sync responses).